apps/wpcom-block-editor/src/default/features/press-this.js

```diff
@@ -0,0 +1,51 @@
+import { createBlock } from '@wordpress/blocks';
+import { dispatch, select, subscribe } from '@wordpress/data';
+import { getQueryArgs } from '@wordpress/url';
+
+const { url, title, text, image, embed } = getQueryArgs( window.location.href );
+
+if ( url ) {
+	const unsubscribe = subscribe( () => {
+		// We need to be sure that the editor is initialized and the core blocks
+		// registered. There is an unstable selector for that, so we use
+		// `isCleanNewPost` otherwise which is triggered when everything is
+		// initialized if the post is new.
+		const editorIsReady = select( 'core/editor' ).__unstableIsEditorReady
```

Comment:
[NIT] We can use [isEditorReady](https://github.com/Automattic/wp-calypso/blob/0ecb6b29ef6241aa938d62ec441e0e7794f5d59f/apps/wpcom-block-editor/src/utils.js#L36) util now!

```js
await isEditorReady();
```

Reply:
nice that is better 👍